cuebot/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Was this eclipse config intentionally added? If not maybe we can add `.project` to the gitignore.

Reply:
no, i'll remove it.